build-config/make/images.make

```diff
@@ -378,7 +405,11 @@ $(RECOVERY_INITRD_STAMP): $(IMAGE_UPDATER_STAMP)
 	$(Q) touch $@
 
 ifeq ($(UEFI_ENABLE),yes)
-  RECOVERY_XORRISO_OPTIONS = $(RECOVERY_CONF_DIR)/xorriso-options-uefi.cfg
+  ifeq ($(ARCH),arm64)
```

Comment:
fixed. Now the bios cfg and uefi cfg are seprated and will be applied based on firmware_type bios or uefi, these both will be merged and applied if firmware_type is selected as auto.